ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -144,33 +150,38 @@ def prepare_lookahead_waypoints(self):
             end_pos = closest_wp_pos + LOOKAHEAD_WPS - 1  # to build waypoint list with a fixed size
             next_waypoints = list(islice(seq, closest_wp_pos, end_pos))  # list of lookahead waypoints
 
-            max_vel = self.mph_to_mps(50)
             if self.traffic is None or self.traffic == -1:
                 self.is_braking = False
```

Comment:
What does self.is_braking do? We assign boolean value to self.is_braking but we do nothing with it afterwards in the code.